gossipd/gossip.c

```diff
@@ -641,7 +641,7 @@ static void handle_get_update(struct peer *peer, const u8 *msg)
 
 reply:
 	msg = towire_gossip_get_update_reply(msg, update);
-	msg_enqueue(&peer->peer_out, take(msg));
+	daemon_conn_send(&peer->owner_conn, take(msg));
```

Comment:
Doesn't this assume that it really is the owner asking for the update? Shouldn't we pass in the `daemon_conn` that the request came from, so we can also handle requests from other daemons?

Reply:
Hmm, good point.